doc/hardware/sensors.rst

```diff
@@ -0,0 +1,41 @@
+
+Sensors list
+##################################
+
+.. table:: Supported sensors on MinnowBoard MAX, Galileo GEN 2 and Edison boards
+
+    ============== ============================================= ==========================================================================
+    Sensor Name     Sensor Type                                    Comment
+    ============== ============================================= ==========================================================================
+    BMA180          Accel                                          missing driver on Edison
+    BMA250          Accel                                          missing driver on Edison
+    LIS3DH          Accel                                          missing driver on Edison
+    MMA7455         Accel                                          missing driver on Edison
+    BMA255          Accel                                          missing driver on Edison
+    LIS2DH12TR      Accel                                          missing driver on Edison
+    BMA222E         Accel                                          missing driver on Edison
+    LSM330DLC       Gyro
+    MPU6050         Gyro
+    MPU9250         Gyro
+    L3G4200         Gyro
+    LSM303          Magnet
```

Comment:
You might also link the sensor name (LSM303) to a page that documents that sensor (such as https://learn.adafruit.com/lsm303-accelerometer-slash-compass-breakout/overview).